app/models/question.rb

```diff
@@ -3,6 +3,8 @@
 class Question < ApplicationRecord
   has_many :answers, dependent: :destroy
 
+  scope :without_answer, -> { Question.left_outer_joins(:answers).where(answers: { id: nil }) }
```

Comment:
you can omit use of Question in the scope - it set ups by default
and also use `joins` over `left_outer_joins`, they are doing the same, but joins looks prettier 
`{ joins(:answers).where(answers: { id: nil }) }`